converters/toZigbee.js

```diff
@@ -849,6 +849,11 @@ const converters = {
             const zclData = {transtime: utils.getTransition(entity, key, meta).time};
 
             if (newColor.rgb !== null || newColor.xy !== null) {
+                if (meta.state.hasOwnProperty('color')) {
+                    delete meta.state.color.hue;
+                    delete meta.state.color.saturation;
+                }
```

Comment:
I'm not sure if this one is acceptable, but let me explain my reasoning.

I have a device that supports both `hs` and `xy`, I

1. set color to `rgb`/`xy`,
2. set color to `hs`.

I'd end up with `{state: color: {x: .., y: .., h: .., s: ..}, color_mode: "hs"}`, I know I can tell which set is correct by looking at `color_mode`, but I find it a bit confusing.

Reply:
Not sure I'd strip them though, as then it would be inconsistent with reporting.
Although if we do, that would remove the need for syncing the values...

Maybe it should have a `color_sync` and `color_strip` device option.

Reply:
Device option or application option?

Reply:
For color_sync I used a device option but ran into some issues with groups... so on the application level might be better IMHO.

I'd expect you generally want all your bulbs to act the same. But either could work, I have no strong preference aside from groups + device_options is a bit trickier as I found out.

Reply:
It would probably be sync or strip... so maybe a `color_processing` options that can take 'strip', 'sync', 'none' as value.

Reply:
dropped the commit on @Koenkk request